README.md

```diff
@@ -1,3 +1,48 @@
 [![Build Status](https://api.travis-ci.org/alphagov/paas-cf.svg)](https://travis-ci.org/alphagov/paas-cf)
 
 # paas-cf
+
+This repository contains [Concourse](http://concourse.ci/) pipelines and related [Terraform](https://terraform.io/) and [BOSH](https://bosh.io/) manifests that allow provisioning of [CloudFoundy](https://www.cloudfoundry.org/) on AWS. It consists of two chains:
+- A concourse pipeline to provision CloudFoundry on AWS using Concourse, Terraform and BOSH
+- A concourse pipeline to destroy previously provisioned environment.
+
+## Operation
```

Comment:
I'm :+1: for the new copy in cce727b and *not* describing these in documentation. They are bound to get out of sync with the actual code.